force-app/main/default/lwc/datatableInlineEditWithUiApi/datatableInlineEditWithUiApi.js

```diff
@@ -0,0 +1,66 @@
+import { LightningElement, wire } from 'lwc';
+import getContacts from '@salesforce/apex/ContactController.getContactList';
+import { refreshApex } from '@salesforce/apex';
+import { updateRecord } from 'lightning/uiRecordApi';
+
+import { ShowToastEvent } from 'lightning/platformShowToastEvent';
+import ID_FIELD from '@salesforce/schema/Contact.Id';
+import FIRSTNAME_FIELD from '@salesforce/schema/Contact.FirstName';
+import LASTNAME_FIELD from '@salesforce/schema/Contact.LastName';
+import TITLE_FIELD from '@salesforce/schema/Contact.Title';
+import PHONE_FIELD from '@salesforce/schema/Contact.Phone';
+import EMAIL_FIELD from '@salesforce/schema/Contact.Email';
+
+const COLS = [
+    { label: 'First Name', fieldName: 'FirstName', editable: true },
```

Comment:
Should be eventually change `fieldName` to be `FIRSTNAME_FIELD.fieldApiName`? To ensure more consistency with static references?